inferno-ml-remote/app/Main.hs

```diff
@@ -0,0 +1,6 @@
+module Main (main) where
```

Comment:
Just a thought, would it make sense to put `inferno-ml-remote` in the same directory and cabal file as `inferno-ml` and just expose it as a separate exe?